js/common.js

```diff
@@ -1043,3 +1046,29 @@ var stopEvent = function(event) {
    event.preventDefault();
    event.stopPropagation();
 };
+
+
+var getGlpiRoot = function() {
```

Comment:
Sounds a bit tricky... What about export a global variable in the HTML page with the php config well known value?